kube-runtime/src/controller/mod.rs

```diff
@@ -267,20 +267,21 @@ where
 /// NOTE: The reason is ignored for comparison purposes. This means that, for example,
 /// an object can only occupy one scheduler slot, even if it has been scheduled for multiple reasons.
 /// In this case, only *the first* reason is stored.
-#[derive(Derivative)]
-#[derivative(
-    Debug(bound = "K::DynamicType: Debug"),
-    Clone(bound = "K::DynamicType: Clone"),
-    PartialEq(bound = "K::DynamicType: PartialEq"),
-    Eq(bound = "K::DynamicType: Eq"),
-    Hash(bound = "K::DynamicType: Hash")
+#[derive(Educe)]
+#[educe(
+    Debug(bound("K::DynamicType: Debug")),
+    Clone(bound("K::DynamicType: Clone")),
+    PartialEq(bound("K::DynamicType: PartialEq")),
```

Comment:
```suggestion
    PartialEq(bound("K::DynamicType: PartialEq")),
    Eq(bound("K::DynamicType: Eq")),
```
Seems to be missing, is there some limitation to use https://github.com/kube-rs/kube/pull/1585/files#diff-d1bc7ffb915e37cefd8621bff0c7c5d9e19965babf0e4b7b02d8da4a8010a871R283 instead?

Otherwise looks good as a drop-in replacement to me, thanks!

Reply:
Yeah, educe doesn't support custom bounds for `Eq`, but I don't think that there is any limitation, except from writing the impl manually.